yarn.lock

```diff
@@ -13956,25 +13956,34 @@ lit-analyzer@^1.2.1:
     vscode-html-languageservice "3.1.0"
     web-component-analyzer "~1.1.1"
 
-lit-element@^3.2.0:
+lit-element@^3.1.0, lit-element@^3.2.0:
```

Comment:
Interesting - why was lit-element@3.1.0 added here when before it was all aligned on 3.2.0?